scripts/west_commands/sbom/README.rst

```diff
@@ -0,0 +1,277 @@
+  ``.c`` files from current directory and all subdirectories recursively, except
+  all ``main.c`` files:
+
+  .. code-block:: bash
+
+     --input-files '**/*.c' '!**/main.c'
+
+* You can read list of input files from a file:
+
+  .. code-block:: bash
+
+     --input-list-file list_file
+
+  It does the same as ``--input-files``, but it reads files and patterns from
+  a file (one file or pattern per line). Files and patterns contained in the
+  list file are relative to the list file location (not the current directory).
+  Comments starting with the ``#`` are allowed.
+
+
+.. _west_sbom Specifying output:
+
+Specifying output
+=================
+
+* You can generate HTML report file:
+
+  .. code-block:: bash
+
+     --output-html <file-name.html>
+
+  If you use ``-d`` option, you can skip this option.
+  The ``sbom_report.html`` file will be generated in your build directory
+  (the first one if you specify more than one build directory).
+
+* Special kind of output is a cache database:
+
+  .. code-block:: bash
+
+     --output-cache-database <cache-database.json>
+
+  For details, see ``cache-database`` detector.
+
+
+.. _west_sbom Detectors:
+
+Detectors
+=========
+
+The SBOM command has following detectors implemented:
+
+* ``spdx-tag`` - search for the ``SPDX-License-Identifier`` in the source code or the binary file.
+  For guidelines, see `SPDX identifier`_. Enabled by default.
+
+* ``full-text`` - compare the contents of the source file with the small database of reference texts.
+  The database is part of the SBOM command. Enabled by default.
+
+* ``scancode-toolkit`` - license detection by the `Scancode-Toolkit`_. Enabled and optional by default.
+
+  If ``scancode`` command is not on your ``PATH``,
+  you can use ``--scancode`` option to provide it, for example:
+
+  .. code-block:: bash
+
+     --scancode ~/scancode-toolkit/scancode
+
+  This detector is optional because is significantly slower than the others.
+
+* ``cache-database`` - use license information detected and cached earlier in the cache database file.
+  Disabled by default.
+
+  You have to provide cache database file with following argument:
+
+  .. code-block:: bash
+
+     --input-cache-database <cache-database.json>
+
+  Each database entry has a path relative to the west workspace directory, a hash and a list of detected licenses.
+  If the file under detection has the same path and hash, then the list of licenses from the database is used.
+
+  .. note::
+     You can generate the database base on e.g scancode-toolkit detector by running following command:
+
+     .. code-block:: bash
+
+        west sbom --input-files <files ...> --license-detectors scancode-toolkit --output-cache-database <cache-database.json>
+
+If you prefer a non-default set of detectors,
+you can provide a list of comma separated detectors with the ``--license-detectors`` option, for example:
+
+  .. code-block:: bash
+
+     --license-detectors spdx-tag,scancode-toolkit
```

Comment:
Does this need a space after the comma?

Reply:
The detector list should be separated only by comma.